docs/source/pages/tutorials/exclude_include_checks.rst

```diff
@@ -0,0 +1,175 @@
+.. Copyright (c) 2024 - 2024, Oracle and/or its affiliates. All rights reserved.
+.. Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+=====================================
+Exclude and include checks in Macaron
+=====================================
+
+.. note::
+
+    This feature is only available from Macaron 0.8.0
+
+This tutorial will show how you can configure Macaron to:
+
+* exclude checks that you don't want to run.
+* include only checks that you want to run.
+
+------------------------------
+Installation and Prerequisites
+------------------------------
+
+This tutorial requires a high-level understanding of checks in Macaron and how they depend on each other. Please see this :ref:`page <macaron-developer-guide>` for more information.
+
+Skip this section if you already know how to install Macaron.
+
+.. toggle::
+
+    Please follow the instructions :ref:`here <installation-guide>`. In summary, you need:
+
+        * Docker
+        * the ``run_macaron.sh``  script to run the Macaron image.
+
+    .. note:: At the moment, Docker alternatives (e.g. podman) are not supported.
+
+
+    You also need to provide Macaron with a GitHub token through the ``GITHUB_TOKEN``  environment variable.
+
+    To obtain a GitHub Token:
+
+    * Go to ``GitHub settings`` → ``Developer Settings`` (at the bottom of the left side pane) → ``Personal Access Tokens`` → ``Fine-grained personal access tokens`` → ``Generate new token``. Give your token a name and an expiry period.
+    * Under ``"Repository access"``, choosing ``"Public Repositories (read-only)"`` should be good enough in most cases.
+
+    Now you should be good to run Macaron. For more details, see the documentation :ref:`here <prepare-github-token>`.
```

Comment:
I don't think we should elaborate on this. If we are worried that the user does not know how to use Macaron, they should not read this page in the first place.

Others may have different ideas about keeping every tutorial to be standalone. I, however, don't think this tutorial should be a standalone one because it introduces an optional configuration rather than a concrete standalone use case. If the user doesn't understand the basic tutorial, I don't expect them to go to this page.

If we are afraid that they find this page by accident for whatever reason, I think it is probably best to say:

```
Prerequisite: You are expected to have gone through <link to the first tutorial>.
```

Otherwise, it just feels too noisy to me.

Reply:
Agreed, this is a tutorial for a feature, not a use case, and so cannot be the user's first encounter with Macaron.

Reply:
Removed in https://github.com/oracle/macaron/pull/667/commits/7da9505dfcd2bd8b0b4e99832939a1701c75c7c8